### DISCUSSION TITLE
Codecov - idea to improve test visibility

### DISCUSSION THREAD

**Discussion by szrtGreg** [DevAdventCalendar/DevAdventCalendar#398] [Ideas] [answered]
Does any one know where is file codecov.yml in our project? According to raport our test coverage is low... howerver I assume that ALL methods are get into calculations. I found interesting option we could ignore some files/methods which are not going to be tested at all. That could increase visibility of real tests. (See URL at the bottom)

![obraz](https://user-images.githubusercontent.com/27915290/94811540-c8765080-03f5-11eb-92a4-61d110c98e14.png)

![obraz](https://user-images.githubusercontent.com/27915290/94811653-f65b9500-03f5-11eb-8111-2ef39380ed4d.png)

https://community.codecov.io/t/100-coverage-with-empty-tests/422

**Comment by WTobor** [comment, marked as answer]
check out file [docker-build.yml](https://github.com/DevAdventCalendar/DevAdventCalendar/blob/develop/.github/workflows/docker-build.yml) - we are already excluding some folders
